### PR TITLE
feat: add support for `upload/get` & `store/get`

### PR DESCRIPTION
**Pull request by olizilla**
- update to `@web3-storage/capabilities@10.2.0`
- update to `@web3-storage/upload-api@6.2.0`
- add `uploadTable.get` implementation
- remove unused deps

License: MIT

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,6 @@
     "@ucanto/validator": "^9.0.0",
     "@web-std/blob": "^3.0.4",
     "@web-std/fetch": "^4.1.0",
-    "@web3-storage/capabilities": "^9.3.0",
```
Comment (olizilla) [inline package.json:35]: I can't see any code at the top level that depends directly on capabilities, so this is removed rather than updated.

File: upload-api/tables/upload.js [hunk 2]
```diff
@@ -104,7 +123,7 @@ export function useUploadTable(dynamoDb, tableName) {
       )
 
       if (!res.Attributes) {
-        throw new Error('Missing `Attributes` property on DyanmoDB response')
+        throw new Error('Missing `Attributes` property on DynamoDB response')
```
Comment (olizilla) [inline upload-api/tables/upload.js:126]: unrelated typo fix.

File: upload-api/tables/store.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export function createStoreTable(region, tableName, options = {}) {
 /**
  * @param {DynamoDBClient} dynamoDb
  * @param {string} tableName
- * @returns {import('@web3-storage/upload-api').StoreTable & import('@web3-storage/upload-api').TestStoreTable}
+ * @returns {import('@web3-storage/upload-api').StoreTable}
```
Comment (olizilla) [inline upload-api/tables/store.js:36]: `TestStoreTable` has gone, now that .get is a thing on `StoreTable`

File: upload-api/tables/upload.js [hunk 4]
```diff
@@ -35,6 +35,25 @@ export function createUploadTable(region, tableName, options = {}) {
...
 export function useUploadTable(dynamoDb, tableName) {
   return {
+    /**
+     * Fetch a single upload
+     *
+     * @param {import('@ucanto/interface').DID} space
+     * @param {import('@web3-storage/upload-api').UnknownLink} root
+     */
+    get: async (space, root) => {
+      console.log('root', root)
```
Comment (alanshaw) [inline upload-api/tables/upload.js:45]:
```suggestion
```

**Review by alanshaw — APPROVED** [review]
👍

**Comment by olizilla** [comment]
@vasco-santos can i get your eyes on this one plz

**Review by vasco-santos — APPROVED** [review]
LGTM
